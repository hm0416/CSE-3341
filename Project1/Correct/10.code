program
	int x;
begin
	x=1-1-1-1;
	output x;
	ref a,b,c;
	a=ref b;
	c=ref a;
end