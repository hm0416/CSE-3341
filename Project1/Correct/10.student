IF
LPAREN
ID[x]
ASSIGN
ASSIGN
ID[y]
RPAREN
THEN
INPUT
ID[x]
SEMICOLON
ELSE
OUTPUT
ID[x]
SEMICOLON

