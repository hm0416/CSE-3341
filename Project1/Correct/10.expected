IF
LPAREN
ID[x]
EQUAL
ID[y]
RPAREN
THEN
INPUT
ID[x]
SEMICOLON
ELSE
OUTPUT
ID[x]
SEMICOLON
